src/transactions/TransactionUtils.h

```diff
@@ -121,6 +133,14 @@ bool isAuthorized(LedgerEntry const& le);
 bool isAuthorized(LedgerTxnEntry const& entry);
 bool isAuthorized(ConstLedgerTxnEntry const& entry);
 
+bool isAuthorizedToMaintainLiabilities(LedgerEntry const& le);
+bool isAuthorizedToMaintainLiabilities(LedgerTxnEntry const& entry);
+bool isAuthorizedToMaintainLiabilities(ConstLedgerTxnEntry const& entry);
+
+bool isAuthorizedForLiabilityDelta(int64_t delta, LedgerTxnEntry const& entry);
+bool expectedAuthorizationMet(TrustLineFlags expectedAuthorization,
```

Comment:
Do you anticipate this being used elsewhere? Otherwise, let's not expose it outside of TransactionUtils.cpp and make it `static` instead.